add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The code directly uses raw SQL queries with user input, which is vulnerable to SQL injection attacks. Additionally, the code is procedural and lacks separation of concerns.

Recommendations:
- **Use Prepared Statements**: Utilize prepared statements to prevent SQL injection. For example, use `mysqli_prepare` and `mysqli_stmt_bind_param`.
- **Refactor to Object-Oriented Design**: Implement a `BookRepository` class to handle database operations. This class can use methods like `addBook`, `deleteBook`, etc.
- **Example**:
  ```php
  class BookRepository {
      private $connection;

      public function __construct($connection) {
          $this->connection = $connection;
      }

      public function addBook($title, $author, $year) {
          $stmt = $this->connection->prepare("INSERT INTO books (title, author, year) VALUES (?, ?, ?)");
          $stmt->bind_param("ssi", $title, $author, $year);
          return $stmt->execute();
      }
  }
  ```
- **Dependency Injection**: Pass the database connection to the `BookRepository` class constructor to adhere to the Dependency Inversion Principle.